apps/desktop/src/lib/components/errorBoundaryActions/InexistentRepo.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+	import RemoveProjectButton from '../RemoveProjectButton.svelte';
+	import { ProjectService } from '$lib/backend/projects';
+	import { getContext } from '$lib/utils/context';
+	import Button from '@gitbutler/ui/Button.svelte';
+
+	const projectService = getContext(ProjectService);
+
+	let isDeleting = $state(false);
+	let deleteSucceeded: boolean | undefined = $state(undefined);
+
+	async function stopTracking() {
+		isDeleting = true;
+		deleteProject: {
+			const id = projectService.getLastOpenedProject();
+			if (id === undefined) {
+				deleteSucceeded = false;
+				break deleteProject;
+			}
+
+			try {
+				await projectService.deleteProject(id);
+			} catch (e) {
+				deleteSucceeded = false;
+				break deleteProject;
+			}
+
+			deleteSucceeded = true;
+		}
+		isDeleting = false;
+	}
+
+	async function locate() {
+		const id = projectService.getLastOpenedProject();
+		if (id === undefined) {
+			return;
+		}
+
+		await projectService.relocateProject(id);
+	}
```

Comment:
I don't love this code, but to me it made a bit more sense to wait on design input before cleaning it up

Reply:
While inexistent is technically a word, it's rarely used. can we name this `NonExsistentRepo` instead? 😏

Reply:
> While inexistent is technically a word
>
😂

Sure thing!